utils.py

```diff
@@ -1,6 +1,17 @@
+ERROR = 'invalid input. try again'
+
+
 def number_validation_input():
     while True:
         try:
             return int(input('enter n: '))
         except ValueError:
-            print('invalid input. try again')
+            print(ERROR)
+
+
+def string_validation_input(prompt):
+    while True:
+        try:
+            return str(input(prompt))
```

Comment:
You don't need type conversion for str. **input()** already returns you str and you can't receive ValueError, so don't need this validation